src/index.html

```diff
@@ -3,107 +3,181 @@
-        <li>Wondering about our other mortgage work? <br />
-          <a class="link-with-icon" href="/mortgage/" target="_blank">Learn how CFPB is protecting mortgage borrowers<span class="cf-icon cf-icon-right"></span></a></li>
-      </ul>
+{% block content %}    
+<main class="content content__2-1 content__bleedbar" id="main" role="main">
+
+    <section class="content_hero">
+      
+      <!-- BACKGROUND IMAGE FOR THE HERO -->
+      <div id="hero-bg-image"></div>
+      
+       <!-- HERO -->
+      <div class="content_wrapper content_no-sidebar">
+          <section class="content_main">
+            <h1>Owning a Home</h1>
+            <h2>We're here to help you with your home-buying process. Learn what to expect and what questions to ask, and get tools and resources to help you make decisions.</h2>
+          </section>
+      </div>
+    </section>
+
+    <div class="content_wrapper">
+        <section class="content_main">
+
+          <section class="section__intro">      
+            <div class="content-l_col content-l_col-1-2">
+              <h2>Know the process</h2>
+              <p>Buying a home can be a daunting process. If you're feeling overwhelmed, you're not alone. We've got tools and resources to help you every step of the way.</p>
+            </div>
+
+            <div class="content-l_col content-l_col-1-2">
+              <img src="http://placekitten.com/380/200" alt="Illustration of the home-buying process">
+          </div>
+          </section>
+
+          <section class="section__phase grid_column__top-divider block__flush-top">
+            <figure class="phase_illu content-l_col content-l_col-1-4">
+              <img src="http://placekitten.com/g/150/150" alt="Illustration of the home-buying process">
+            </figure>
+
+            <div class="phase_intro content-l_col content-l_col-1-2">
+              <div class="content-l_col content-l_col-1 content-l__large-gutters">
+              <h3>Prepare to shop</h3>
+              <p>Not sure how to get started, how much you can afford, or what to expect when buying a home? Set yourself up for success with a little bit of preparation.</p>
+              <p><a class="go-link" href="{{ base_url }}process/prepare">Learn more</a></p>
+              </div>
+            </div>
+
+             <div class="phase_tools content-l_col content-l_col-1-4 block__sub grid_column__left-divider">
+                <h6 class="block__sub block__flush-top">Key tools</h6>
+                <ul class="list__unstyled block__flush-top">
+                  <li class="block__sub block__flush-top"><a href="{{ base_url }}prepare-worksheets">Worksheet: clarify your goals</a></li>
+                  <li class="block__sub block__flush-top"><a href="{{ base_url }}check-rates">Check interest rates for your situation</a></li>
+                </ul>
+              </div>
+          </section>
+
+          <section class="section__phase grid_column__top-divider block__flush-top">
```

Comment:
Instead of the top-divider & flush-top classes, we can use `block__border-top block__padded-top block__sub` on the section to get the border/padding.